WebRoot/js/zimbraMail/calendar/view/ZmLocationSuggestionView.js

```diff
@@ -105,6 +105,7 @@ function(itemDiv, ev) {
 ZmLocationSuggestionView.prototype._setNoResultsHtml =
 function() {
     var	div = document.createElement("div");
```

Comment:
I think there's a tab between "var" & "div" , maybe we can change it to space with this change ?

Reply:
Fixed: https://github.com/Zimbra/zm-web-client/pull/154/commits/ed2bb21d4bca8573e9059d8e9694e9b0316b417a

Thanks